parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -24,18 +24,40 @@ write_files:
     ExecStart=
     ExecStart=/usr/bin/docker daemon -H fd:// --storage-driver=overlay2 --bip={{WrapAsVariable "dockerBridgeCidr"}}
 
+{{if and (IsNSeriesSKU .) (UseNvidiaRuntime)}}
```

Comment:
Can we move this `{{if}}` inside the content spec? (both conditions scaffold the same file/props)